evo-delivery-app/evo-delivery-backend/src/configs/ea-constants.ts

```diff
@@ -0,0 +1,2 @@
+export const EA_ENGINE_BASE_URL = `http://127.0.0.1:5000`;
```

Comment:
maybe add _LOCAL suffix, or we can move it to be /env var later on

Reply:
done

Reply:
we still need to find a way to do it dynamically...
I have added an issue for that -> #61